### ISSUE TITLE
rabbitmq crashes

### ISSUE THREAD

**Issue by kmeaw** [kirichenko-o/md5-file-hasher#3]
```
% docker-compose logs --no-log-prefix -t rabbitmq
Attaching to md5-file-hasher_rabbitmq_1
2021-07-26T21:34:00.527684098Z error: RABBITMQ_DEFAULT_PASS is set but deprecated
2021-07-26T21:34:00.527737423Z error: RABBITMQ_DEFAULT_USER is set but deprecated
2021-07-26T21:34:00.527750910Z error: RABBITMQ_DEFAULT_VHOST is set but deprecated
2021-07-26T21:34:00.527944161Z error: deprecated environment variables detected
2021-07-26T21:34:00.527962174Z 
2021-07-26T21:34:00.527986233Z Please use a configuration file instead; visit https://www.rabbitmq.com/configure.html to learn more
2021-07-26T21:34:00.527999297Z 
```

**Comment by kmeaw** [comment]
```
% docker inspect -f '{{.Id}} {{.RepoTags}} {{.RepoDigests}}' rabbitmq:latest
sha256:0deab9453932c066db701c8093f77a92588febf6348d796f6c8acb17a92c7273 [rabbitmq:latest] [rabbitmq@sha256:1ff56ae86daa8606caf84f4eb367cff72d5e748f86f6421b58fa714a10acc142]
```

**Comment by kmeaw** [comment]
Creating rabbitmq users manually (using rabbitmqctl) instead of setting environment variables does the trick.
Even file upload seem to succeed and calculated md5 are fine, however every upload causes an erlang process to crash: https://gist.github.com/kmeaw/f2f37219daeb3034c0e77567004880b0

**Comment by kmeaw** [comment]
This issue seems to be fixed in eb93ef68a21f8f63d9960ab7f229ed874f7a8854, downgrading rabbitmq image from 3.9.0 to 3.8.19 no longer produces the crash.
